tests/kuttl/common/assert_keystone_deployment.yaml

```diff
@@ -0,0 +1,134 @@
+#
```

Comment:
should we name this assert_sample.yaml or something and have this common in all operators so that it can be referenced from install_yamls validation target?

Reply:
Done